pkg/cache/cache_unit_test.go

```diff
@@ -93,73 +93,73 @@ var _ = Describe("cache.inheritFrom", func() {
 	})
 	Context("Resync", func() {
 		It("is nil when specified and inherited are unset", func() {
-			Expect(checkError(specified.inheritFrom(inherited)).Resync).To(BeNil())
+			Expect(checkError(specified.inheritFrom(inherited)).ResyncEvery).To(BeNil())
 		})
 		It("is unchanged when only specified is set", func() {
-			specified.Resync = pointer.Duration(time.Second)
-			Expect(checkError(specified.inheritFrom(inherited)).Resync).To(Equal(specified.Resync))
+			specified.ResyncEvery = pointer.Duration(time.Second)
+			Expect(checkError(specified.inheritFrom(inherited)).ResyncEvery).To(Equal(specified.ResyncEvery))
 		})
 		It("is inherited when only inherited is set", func() {
-			inherited.Resync = pointer.Duration(time.Second)
-			Expect(checkError(specified.inheritFrom(inherited)).Resync).To(Equal(inherited.Resync))
+			inherited.ResyncEvery = pointer.Duration(time.Second)
+			Expect(checkError(specified.inheritFrom(inherited)).ResyncEvery).To(Equal(inherited.ResyncEvery))
 		})
 		It("is unchanged when both inherited and specified are set", func() {
-			specified.Resync = pointer.Duration(time.Second)
-			inherited.Resync = pointer.Duration(time.Minute)
-			Expect(checkError(specified.inheritFrom(inherited)).Resync).To(Equal(specified.Resync))
+			specified.ResyncEvery = pointer.Duration(time.Second)
+			inherited.ResyncEvery = pointer.Duration(time.Minute)
+			Expect(checkError(specified.inheritFrom(inherited)).ResyncEvery).To(Equal(specified.ResyncEvery))
 		})
 	})
 	Context("Namespace", func() {
 		It("is NamespaceAll when specified and inherited are unset", func() {
```

Comment:
Either the title or the validation is not correct anymore

Reply:
fixed, thanks!